vnext/Microsoft.ReactNative/Views/TouchEventHandler.cpp

```diff
@@ -427,7 +427,7 @@ bool TouchEventHandler::DispatchBackEvent() {
     return false;
 
```

Comment:
We should probably be checking the QuirkSettings::GetBackHandlerKind before forwarding the event to JS and marking it as handled. Looks like we are not correctly checking that in the case of the pointer event.

See https://microsoft.github.io/react-native-windows/docs/native-api/QuirkSettings#setbackhandlerkind

```cpp
if (winrt::Microsoft::ReactNative::implementation::QuirkSettings::GetBackHandlerKind(
          winrt::Microsoft::ReactNative::ReactPropertyBag(m_context->Properties())) ==
      winrt::Microsoft::ReactNative::BackNavigationHandlerKind::JavaScript)
```